### ISSUE TITLE
'az network application-gateway waf-policy managed-rule rule-set add' cannot set OWASP 3.1

### ISSUE THREAD

**Issue by marlowp** [Azure/azure-cli#14360]
**Describe the bug**
When using the Azure CLI to set the managed rule set it is not possible to set it to OWASP 3.1. It always defaults to OWASP 3.0.

**To Reproduce**
Create a waf-policy via:
```powershell
az network application-gateway waf-policy create `
    --name "waf-policy-test" `
    --resource-group "rg-test"
```

Then add a managed rule via:
```powershell
az network application-gateway waf-policy managed-rule rule-set add `
    --policy-name "waf-policy-test" `
    --resource-group "rg-test" `
    --type "OWASP" `
    --version 3.1
```

**Expected behavior**
The managed rule set should be set to OWASP_3.1 but is instead set to OWASP_3.0

**Environment summary**
azure-cli - 2.9.0

command-modules-nspkg - 2.0.3
core - 2.9.0
nspkg- 3.0.4
telemetry - 1.0.4

Extensions:
azure-devops - 0.18.0

**Additional context**
Similar to the issue raised [here](https://github.com/Azure/azure-cli/issues/12387), which has been closed.

**Comment by yonzhan** [comment]
add to S174

**Comment by MyronFanQiu** [comment]
@marlowp Hi. The first managedRuleSets is always 3.0. You can add more 3.1 0.1 managedRuleSets into the waf policy. Service team need a mandatory rule set when we create waf policy. Is it a blocker for you?

**Comment by alexpilon666** [comment]
@MyronFanQiu , flagged a similar-enough issue yesterday: #14499 
TL;DR: Right now, with Azure CLI, there's no way to set a newly-created WAF Policy to OWASP 3.1 or even 2.2.9.
You can easily add the `Microsoft_BotManagerRuleSet` v0.1 on top of OWASP, but since you can't remove OWASP nor add a second version, and the `update` action does nothing, we're screwed.
